cmd/util/ledger/migrations/change_contract_code_migration.go

```diff
@@ -331,16 +330,30 @@ func SystemContractChanges(chainID flow.ChainID) []SystemContractChange {
 			systemContracts.ViewResolver,
 			nftContracts.ViewResolver(),
 		),
+	}
 
-		// EVM related contracts
-		NewSystemContractChange(
-			systemContracts.EVMContract,
-			evm.ContractCode(
-				systemContracts.FlowToken.Address,
-				true,
+	if chainID != flow.Emulator {
+		contractChanges = append(contractChanges,
+			SystemContractChange{
+				Address:      fungibleTokenSwitchboardAddress,
+				ContractName: "FungibleTokenSwitchboard",
+				NewContractCode: string(ftContracts.FungibleTokenSwitchboard(
+					systemContracts.FungibleToken.Address.HexWithPrefix(),
+				)),
+			},
+
+			// EVM related contracts
+			NewSystemContractChange(
+				systemContracts.EVMContract,
+				evm.ContractCode(
+					systemContracts.FlowToken.Address,
+					true,
+				),
```

Comment:
maybe this is working now after https://github.com/onflow/cadence-tools/pull/280